opal/corelib/enumerable.rb

```diff
@@ -177,6 +177,10 @@ def count(object = undefined, &block)
   end
 
   def cycle(n = nil, &block)
+    if `arguments.length > 1`
```

Comment:
The arity check, when compiled in for the cycle method, looks weird:

```js
var $arity = arguments.length;if ($arity < 0) { Opal.ac($arity, -1, this, "cycle"); }
```

How could the args array ever have a length < 0?